pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -299,9 +299,6 @@ func (r *ReconcileNode) reconcileCrashRetention(namespace string, cephCluster ce
 		var cronJob client.Object
 		// minimum k8s version required for v1 cronJob is 'v1.21.0'. Apply v1 if k8s version is at least 'v1.21.0', else apply v1beta1 cronJob.
 		if useCronJobV1 {
-			if err := r.deletev1betaJob(objectMeta); err != nil {
```

Comment:
we don't in this code, This is the deletion part of the code and we do not want to delete v1beta1 while deleting v1 since we already deleting the v1beta1 cronJob while creating v1 CronJob. @travisn

Reply:
Aha, I see it now, we really don't need this since during upgrade it is deleted elsewhere. And since the v1 job is deleted, we will also attempt to delete the v1beta1 job here.